### PR TITLE
Fix / Car switching

### PR DESCRIPTION
**Pull request by amandaflod**
## Describe your changes

Added SelfSwitcherBubble with info text if banksignering is not possible. 

https://user-images.githubusercontent.com/59652803/221874781-94602311-b62d-47b4-ac8b-b2babdb358cb.mov

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/SelfSwitcherBubble.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  backgroundColor: theme.colors.blueFill1,
+  paddingBlock: theme.space.sm,
+  paddingRight: theme.space.md,
+  paddingLeft: '2.25rem',
+  borderRadius: theme.radius.sm,
+})
+
+const StyledInfoIcon = styled(InfoIcon)({
+  position: 'absolute',
+  top: '0.875rem',
```
Comment (robinandeer) [inline apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/SelfSwitcherBubble.tsx:30]: No theme value available?

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx [hunk 2]
```diff
@@ -56,7 +58,9 @@ export const CancellationForm = ({ option, ...props }: Props) => {
 type NoCancellationProps = Pick<Props, 'onStartDateChange' | 'startDate'>
 
 const NoCancellation = ({ onStartDateChange, startDate, ...props }: NoCancellationProps) => {
-  return <SmartDateInput {...props} date={startDate} onChange={onStartDateChange} />
+  const date = startDate ?? new Date()
```
Comment (robinandeer) [inline apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx:61]: I think we want the API to control the default date value...no?

**Review by robinandeer — COMMENTED** [review]
Just some comments - will review when back to laptop

File: apps/store/src/utils/formatter.ts [hunk 3]
```diff
@@ -44,6 +44,16 @@ export const formatDateFromNow = (date: Date, options: DateFormatOptions): strin
   return date.toLocaleDateString(options.locale)
 }
 
+export const formatTitleCase = (str: string): string => {
```
[the remark this answers is not in the document]
Comment (amandaflod) [reply apps/store/src/utils/formatter.ts:47]: I wanted to, but as far as I know it's not that simple to change allcaps to titlecase with css :/

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/SelfSwitcherBubble.tsx (hunk 1 above)
Comment (amandaflod) [reply apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/SelfSwitcherBubble.tsx:30]: No, unfortunately not :/

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/SelfSwitcherBubble.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+export const SelfSwitcherBubble = ({ date }: { date: Date }) => {
+  const { t } = useTranslation('purchase-form')
+  const formatter = useFormatter()
+
+  const formattedDate = formatter.fromNow(date)
+  return (
+    <Wrapper>
+      <StyledInfoIcon color={theme.colors.blue600} />
+      <Text size="xs"> {t('SELF_SWICHER_MESSAGE', { date: formattedDate })}</Text>
```
[the remark this answers is not in the document]
Comment (amandaflod) [reply apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/SelfSwitcherBubble.tsx:14]: woopsie 🙃

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx (hunk 2 above)
Comment (amandaflod) [reply apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx:61]: The default value is null, but in most cases (all except banksignering) we want it to be set to today. Maybe there is a better solution but went with this for now :))

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx [hunk 5]
```diff
@@ -95,32 +100,31 @@ type BankSigneringCancellationProps = Pick<
 }
 
 const BankSigneringCancellation = (props: BankSigneringCancellationProps) => {
-  const { onAutoSwitchChange, companyName, requested, onStartDateChange } = props
+  const { onAutoSwitchChange, companyName, requested, onStartDateChange, startDate } = props
   const { t } = useTranslation('purchase-form')
+  const formatter = useFormatter()
+
+  const formattedCompanyName = formatter.titleCase(companyName)
+  const date = startDate ?? new Date()
```
[the remark this answers is not in the document]
Comment (amandaflod) [reply apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx:108]: Good catch! Will fix

File: apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx (hunk 2 above)
Comment (robinandeer) [reply apps/store/src/components/ProductPage/PurchaseForm/CancellationForm/CancellationForm.tsx:61]: Right right, let's review after launch and make sure backend can own this

**Review by robinandeer — APPROVED** [review]
Looks good!

**Pull request merged**